### PR TITLE
Docs: update AC info in API 2.0 migration guide

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *update info regarding AC IR v 10 support*
 - *...*

### Tickets:
 - *80841*

### CODE REVIEW

File: docs/OV_Runtime_UG/migration_ov_2_0/intro.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The IR v11 is supported by all OpenVINO Development tools including Post-Trainin
 OpenVINO API 2.0 also supports models in IR v10 for backward compatibility. So, if a user has an IR v10, it can be fed to OpenVINO Runtime as well (see [migration steps](common_inference_pipeline.md)).
 
 Some OpenVINO Development Tools also support both IR v10 and IR v11 as an input:
-- Accuracy checker also supports IR v10, but requires an additional option to denote which API is used underneath.
+- Accuracy checker uses API 2.0 for models accuracy measurement by default, but also supports switching to API 1.0  using command line parameter `--use_new_api False`. Both launchers accepts IR v10 and v11, but in some cases configuration files should be updated. More details can be found in [Accuracy Checker documentation](https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/openvino_launcher_readme.md#migrate-accuracy-checker-configuration-from-openvino-api-10-to-20).
```
Comment (eaidova) [inline docs/OV_Runtime_UG/migration_ov_2_0/intro.md:44]:
```suggestion
- Accuracy checker uses API 2.0 for models accuracy measurement by default, but also supports switching to API 1.0  using command line parameter `--use_new_api False`. Both launchers accept IR v10 and v11, but in some cases configuration files should be updated. More details can be found in [Accuracy Checker documentation](https://github.com/openvinotoolkit/open_model_zoo/blob/releases/2022/1/tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/openvino_launcher_readme.md).
```

**Review by ilyachur — APPROVED** [review]

File: docs/OV_Runtime_UG/migration_ov_2_0/intro.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ The IR v11 is supported by all OpenVINO Development tools including Post-Trainin
 OpenVINO API 2.0 also supports models in IR v10 for backward compatibility. So, if a user has an IR v10, it can be fed to OpenVINO Runtime as well (see [migration steps](common_inference_pipeline.md)).
 
 Some OpenVINO Development Tools also support both IR v10 and IR v11 as an input:
-- Accuracy checker also supports IR v10, but requires an additional option to denote which API is used underneath.
+- Accuracy checker uses API 2.0 for models accuracy measurement by default, but also supports switching to API 1.0 using command line parameter `--use_new_api False`. Both launchers accept IR v10 and v11, but in some cases configuration files should be updated. More details can be found in [Accuracy Checker documentation](https://github.com/openvinotoolkit/open_model_zoo/blob/releases/2022/1/tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/openvino_launcher_readme.md).
```
Comment (kblaszczak-intel) [inline docs/OV_Runtime_UG/migration_ov_2_0/intro.md:44]:
```suggestion
- Accuracy checker uses API 2.0 for model accuracy measurement by default, but also supports switching to API 1.0 using the `--use_new_api False` command line parameter. Both launchers accept IR v10 and v11, but in some cases configuration files should be updated. More details can be found in [Accuracy Checker documentation](https://github.com/openvinotoolkit/open_model_zoo/blob/releases/2022/1/tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/openvino_launcher_readme.md).
```

Comment (eaidova) [reply docs/OV_Runtime_UG/migration_ov_2_0/intro.md]: applied, thanks

Comment (kblaszczak-intel) [reply docs/OV_Runtime_UG/migration_ov_2_0/intro.md]: A few other changes need to be included here, like the title cannot state "OpenVINO™ 2.0." I know @xu-yuan1 has already made some suggestions in a separate PR.

**Pull request merged**